infrastructure/resource_group_level/application-gateway-portal.bicep

```diff
@@ -0,0 +1,287 @@
+param location string
+param environment string
+param gatewaySubnetId string
+param tfsIpId string
+param portalApiHostname string
+param portalUiHostname string
+param gefUiHostname string
+param apiPortalAccessPort int = 0
```

Comment:
Note that the Api portal seems to only be for `dev`. I've adjusted this and the NSG code to only set it up if has been given a non-zero value. (e.g. `44232`.)